src/firecracker/swagger/firecracker.yaml

```diff
@@ -282,6 +282,30 @@ paths:
           schema:
             $ref: "#/definitions/Error"
 
+  /hotplug:
+    put:
+      summary: Used to hot plug resources into the microVM
+      description:
+        Adds new resources to the VM. Only available for x86_64 hosts.
+      parameters:
+        - name: body
+          in: body
+          description: The resource type and amount to be hotplugged.
+          required: true
+          schema: 
+            $ref: "#/definitions/Hotplug"
+      responses:
+        204:
+          description: Resources hotplugged successfully.
+        400: 
+          description: Resources cannot be hotplugged due to bad input.
```

Comment:
Can the errors be only due to bad input?

Reply:
In that case, I suggest we update the description here.

Reply:
Will do

Reply:
Updated now. Decided to leave it just as "bad input or VMM error", as I imagine users are not interested in the specific VMM-related errors that could occur.